commands/ps/index.js

```diff
@@ -41,6 +29,41 @@ function printExtended (dynos) {
   })
 }
 
+function * printAccountQuota (heroku, app) {
+  if (app.process_tier !== 'free') {
+    return
+  }
+
+  let account = yield heroku.request({path: '/account'})
```

Comment:
I missed that `account.id` somehow when I was looking at this before. You're right they could be bundled, and I think client-side that would be preferable (an extra request doesn't really affect us), but it might make the code a bit ugly. I could go either way too.